consul/parameters/init/sysvinit.yaml

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+#
+# Set default values.
+---
+values:
+  service_path: /etc/init.d/consul
```

Comment:
This file appears to be superfluous since this value is already provided in `defaults.yaml`.  Or perhaps it's `defaults.yaml` that should be modified instead, say to use `systemd` values as the default values and then remove that file instead.

Reply:
Ok for using systemd by default, it's the most used `init` system nowadays